tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -144,13 +144,16 @@ def __init__(self, marionette):
         update = Wait(self.marionette).until(expected.element_present(*self._update_locator))
         Wait(self.marionette).until(lambda m: update.location['y'] == 0 and update.is_displayed())
 
-    def tap_update(self, return_details=True):
+    def tap_update(self, return_details=True, return_merge=False):
```

Comment:
Having 2 boolean attributes doesn't look okay. What is the expected behavior if, as a test writer, I put: `return_details=True` and `return_merge=True`?

I'd change it to something like:
```py
DETAILS_PANEL = 'details'
MERGE_PANEL = 'merge'

def tap_update(self, return_panel=True):
    # ...
    if return_panel == self.DETAILS_PANEL:
        # ...
    elif return_panel == self.MERGE_PANEL:
        # ...
    else:
        # ...
```
And then you call it this way:
``` py
contacts_form.tap_update(return_panel=contacts_form.MERGE_PANEL)
```